src/drivers/camera_trigger/camera_trigger.cpp

```diff
@@ -503,6 +534,8 @@ CameraTrigger::test()
 	vcmd.timestamp = hrt_absolute_time();
 	vcmd.param5 = 1.0;
 	vcmd.command = vehicle_command_s::VEHICLE_CMD_DO_DIGICAM_CONTROL;
+	vcmd.target_system = 1;
```

Comment:
You might want to take this from `vehicle_status`.

Reply:
I actually store and keep the target_system and target_component for the new solution when it receives a valid command.

I just added this at the test function as it was not working, these would be the standard values, and testing can happen despite receiving a valid command. Could you elaborate on how to get this information from vehicle_status?